src/collectd.conf.pod

```diff
@@ -5131,13 +5131,12 @@ The B<Query> block defines one database query which may later be used by a
 database definition. It accepts a single mandatory argument which specifies
 the name of the query. The names of all queries have to be unique (see the
 B<MinVersion> and B<MaxVersion> options below for an exception to this
-rule). The following configuration options are available to define the query:
+rule).
 
-In each B<Query> block, there is one or more B<Result> blocks. B<Result>
-blocks define how to handle the values returned from the query. They define
-which column holds which value and how to dispatch that value to the daemon.
-Multiple B<Result> blocks may be used to extract multiple values from a single
-query.
+In each B<Query> block, there is one or more B<Result> blocks. Multiple
```

Comment:
I do not understand. s/is/are/ ???

Copy-n-pasted from line 1872:

cat collectd.conf.pod |grep -n "there is one or more"
1872:In each B<Query> block, there is one or more B<Result> blocks. B<Result> blocks
5136:In each B<Query> block, there is one or more B<Result> blocks. Multiple